config/jail.conf

```diff
@@ -344,7 +344,7 @@ logpath = %(lighttpd_error_log)s
 [roundcube-auth]
 
 port     = http,https
-logpath  = /var/log/roundcube/userlogins
+logpath  = /var/log/roundcubemail/errors
```

Comment:
Are you referring to the directory /var/log/roundcube being changed to /var/log/roundcubemail? Or the file "userlogins" being changed to "errors"?